WooCommerce/Classes/POS/Presentation/CardReaderConnection/UI States/PointOfSaleUnsupportedWidthView.swift

```diff
@@ -10,7 +10,7 @@ struct PointOfSaleUnsupportedWidthView: View {
             VStack(alignment: .center, spacing: PointOfSaleItemListErrorLayout.headerSpacing) {
                 Image(systemName: "ipad.landscape.badge.exclamationmark")
                     .symbolRenderingMode(.palette)
-                    .foregroundStyle(Color(.wooCommerceAmber(.shade60)), Color.posPrimaryText)
+                    .foregroundStyle(Color(.wooCommerceAmber(.shade60)), Color.posOnSurface)
```

Comment:
Nit: We can use `posAlert` in substitution of `wooCommerceAmber`

Reply:
Nice catch, I made the update in https://github.com/woocommerce/woocommerce-ios/pull/15119/commits/fbbc6350c2bdff141fbbaafbaa003fbaba99732c along with 2 other cases.

<img width="640" alt="Screenshot 2025-02-13 at 9 05 49 AM" src="https://github.com/user-attachments/assets/4fdd02a8-23dc-4688-bab7-440d2934f800" />
<img width="646" alt="Screenshot 2025-02-13 at 9 06 14 AM" src="https://github.com/user-attachments/assets/94adb42b-a10d-4b92-8d10-aefe2faa49dc" />
<img width="941" alt="Screenshot 2025-02-13 at 9 07 04 AM" src="https://github.com/user-attachments/assets/1145b44b-61fa-4a4c-949e-e87ecda9677a" />